static/js/containers/CreatePostPage.js

```diff
@@ -17,12 +18,16 @@ import type { Match } from "react-router"
 type CreatePostPageProps = {
   match: Match,
   dispatch: Dispatch,
-  postForm: FormValue,
+  postForm: ?FormValue,
   channel: Channel,
   channels: RestState<Array<Channel>>,
   history: Object
 }
 
+const CREATE_POST_KEY = "post:new"
+const CREATE_POST_PAYLOAD = { key: CREATE_POST_KEY }
+const getForm = R.prop(CREATE_POST_KEY)
```

Comment:
We'll eventually have a different form to edit a post (editable fields are limited to `text`), so I want to push back a tad on the namespacing because I think it's  important to have deterministic keys that represent distinct form data structures. For the same reason I think the defs of those values should stay here because they're only pertinent to the `CreatePostPage`.